src/main/java/teammates/ui/webapi/action/Action.java

```diff
@@ -190,6 +193,14 @@ protected String getRequestBody() {
         return requestBody;
     }
 
+    protected FeedbackSessionAttributes getFeedbackSession(String feedbackSessionName, String courseId) {
```

Comment:
Just `getFeedbackSession` doesn't immediately tell that it's going to throw some exception if the value is null. Name it something like `getNonNullFeedbackSession`?